tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/LoginSteps.cs

```diff
@@ -8,18 +11,25 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
     [Binding]
     internal sealed class LoginSteps
     {
+        private readonly Dictionary<string, string> HostUrls = new Dictionary<string, string>
```

Comment:
After the changes I don't think this is required anymore. Can you review this please?